### PR TITLE
E2E Regression basic messages

### PR DESCRIPTION
**Pull request by rajmundex**
Jira: [CP-XXX]

E2E tests for Messages
- updated closing modal method which may appear after after launching app 
- added required page objects+methods for messages
- sending message to incorrect number 
    - using send button
    - using Enter key
- deleting thread using delete button on open thread screen
- deleting thread by deleting last message in the thread
- deleting thread from the thread list screen
- deleting thread using selection manager

<details>
<summary><b>Screenshots</b></summary>
image
</details>

### CODE REVIEW

File: packages/e2e/src/page-objects/messages.page.ts [hunk 1]
```diff
@@ -19,23 +19,220 @@ class MessagesPage extends Page {
...
+    await this.containerMessageContent.moveTo()
+  }
+
+  public get messageOptionsDropdownList(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="message-bubble-dropdown-action-button"]')
+  }
+
+  //
```
Comment (lkowalczyk87) [inline packages/e2e/src/page-objects/messages.page.ts:99]: :)

File: packages/e2e/src/page-objects/messages.page.ts [hunk 2]
```diff
@@ -19,23 +19,220 @@ class MessagesPage extends Page {
...
+    await this.insertTextToMessageInput(messageText)
+    //send by pressing Send button
+
+    await this.sendMessageButton.waitForClickable()
+    await this.sendMessageButton.click()
+  }
+
+  async clickCloseThreadDetailsScreen() {
+    await this.closeButtonOnThreadDetailScreen.waitForClickable({
+      timeout: 7000,
```
Comment (lkowalczyk87) [inline packages/e2e/src/page-objects/messages.page.ts:178]: 🟢 I noticed that timeout is usually set to 7000 ms. You might want to move the timeout value to the dedicated const.

**Review by lkowalczyk87 — APPROVED** [review]

File: packages/e2e/src/page-objects/messages.page.ts [hunk 3]
```diff
@@ -19,23 +19,220 @@ class MessagesPage extends Page {
...
+    return $('[data-testid="messages-empty-thread-list-state"]')
+  }
+
+  public get threadScreenThreadDropdownOptions(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="dropdown"]')
+  }
+
+  public get deleteOnThreadDropdownOptions(): ChainablePromiseElement<
```
Comment (dkarski) [inline packages/e2e/src/page-objects/messages.page.ts:123]: this is a list or a element of the options list? 🤔 
```suggestion
  public get deleteOnThreadDropdownOption(): ChainablePromiseElement<
```

**Review by dkarski — APPROVED** [review]
I have a few doubts about name conventions. Will be easier for me to discuss this in a live session. Today I can't so I don't block this PR. We could talk about it on a separate pull request.

**Review by dkarski — APPROVED** [review]
LGTM ✅

**Pull request merged**